lib/ui/painting/image_encoding_impeller.cc

```diff
@@ -74,19 +77,21 @@ void DoConvertImageToRasterImpeller(
 
 void ImageEncodingImpeller::ConvertDlImageToSkImage(
     const sk_sp<DlImage>& dl_image,
-    std::function<void(sk_sp<SkImage>)> encode_task,
+    std::function<void(fml::StatusOr<sk_sp<SkImage>>)> encode_task,
     const std::shared_ptr<impeller::Context>& impeller_context) {
   auto texture = dl_image->impeller_texture();
 
   if (impeller_context == nullptr) {
     FML_LOG(ERROR) << "Impeller context was null.";
```

Comment:
Do we need the FML_LOG(ERROR) if we're surfacing the exact error message now?

Reply:
removed